src/views/Home/Home.jsx

```diff
@@ -1,17 +1,26 @@
-import React, { useEffect } from 'react';
-import { useLocation } from 'react-router-dom';
+import React, { useEffect, useState } from 'react';
 
 import NoResults from '../../components/molecules/no-results/no-results';
-import { getKeyValueFromQueryString } from '../../utils';
+import SearchResultsList from '../../components/molecules/search-results-list/search-results-list';
+
+import { useURLQuery, useCustomQuery } from '../../hooks';
+import { getSearchResults } from '../../services/api/actions/';
 import { useStateValue } from '../../store/store';
 import { useTracking } from 'react-tracking';
 
 const Home = () => {
-	const location = useLocation();
-	const { search } = location;
-	const [{ canonicalHost, title, siteName }] = useStateValue();
-	const keyword = getKeyValueFromQueryString('swKeyword', search);
+	const urlQuery = useURLQuery();
+	const [{ canonicalHost, siteName, title }] = useStateValue();
+	const [ definitionResult, setDefinitionResult ] = useState();
```

Comment:
updating to **stateSearchResults** & **setSearchResult**

Reply:
well, it would be stateSearchResults, setStateSearchResults